vehicles/src/modules/permit/application.service.ts

```diff
@@ -272,6 +272,7 @@ export class ApplicationService {
       unitNumber: 'permitData.unitNumber',
       plate: 'permitData.plate',
       applicant: 'permitData.applicant',
+      updatedDateTime: 'permit.updatedDateTime'
```

Comment:
Already taken up as part of https://github.com/bcgov/onroutebc/pull/1244. Waiting for @zgong-gov FE changes for the PR to be merged. Kindly coordinate with @zgong-gov